### PR TITLE
feat(icon): create login and storage icon

### PR DESCRIPTION
**Pull request by indiramuru**
IconLogin:
Screen Shot 2019-11-05 at 11 19 22 AM
IconStorage:
Screen Shot 2019-11-05 at 3 18 57 PM

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @indiramuru has signed the CLA. Thanks for the pull request!

File: src/icons/general/IconLogin.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    color?: string,
+    height?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconLogin = ({ className = '', color = bdlGray80, height = 16, title, width = 16 }: Props) => (
+    <AccessibleSVG
+        className={`icon-login ${className}`}
```
Comment (priyajeet) [inline src/icons/general/IconLogin.js]: bdl / SUIT style

File: src/icons/general/IconStorage.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    className?: string,
+    color?: string,
+    height?: number,
+    title?: string,
+    width?: number,
+};
+
+const IconStorage = ({ className = '', color = '#3FB87F', height = 12, title, width = 16 }: Props) => (
+    <AccessibleSVG
+        className={`icon-storage ${className}`}
```
Comment (priyajeet) [inline src/icons/general/IconStorage.js]: likewise

File: src/icons/general/IconStorage.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    title?: string,
+    width?: number,
+};
+
+const IconStorage = ({ className = '', color = '#3FB87F', height = 12, title, width = 16 }: Props) => (
```
Comment (priyajeet) [inline src/icons/general/IconStorage.js]: proper color

Comment (indiramuru) [reply src/icons/general/IconStorage.js]: do you mean to use any of the color from `/styles/variables.js`

Comment (alexkrolick) [reply src/icons/general/IconStorage.js]: Ask the designer to review the palette

Comment (indiramuru) [reply src/icons/general/IconStorage.js]: yes checking with her. thanks

Comment (indiramuru) [reply src/icons/general/IconStorage.js]: updated the PR with bdlGreenLight

File: src/icons/general/__tests__/IconStorage.test.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import IconStorage from '../IconStorage';
+
+describe('icons/general/IconAlignLeft', () => {
```
Comment (jstoffan) [inline src/icons/general/__tests__/IconStorage.test.js]: Should this be `icons/general/IconStorage`, as well?

File: src/icons/general/__tests__/IconStorage.test.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
```
Comment (jstoffan) [inline src/icons/general/__tests__/IconStorage.test.js]: Whitespace between imports should be removed.

**Comment by indiramuru** [comment]
@jstoffan addressed your comments

File: src/icons/general/IconStorage.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        className={`bdl-IconStorage ${className}`}
+        height={height}
+        title={title}
+        viewBox="0 0 16 12"
+        width={width}
+    >
+        <rect
+            className="background-color"
+            clipRule="evenodd"
+            fill="#D8D8D8"
```
Comment (priyajeet) [inline src/icons/general/IconStorage.js]: hardcoded?

**Pull request merged**